packages/ui-components/src/components/tag-letter/tag-letter.tsx

```diff
@@ -0,0 +1,28 @@
+import { Component, Host, h, Prop } from '@stencil/core';
+
+@Component({
+	tag: 'kv-tag-letter',
+	styleUrl: 'tag-letter.scss',
+	shadow: true
+})
+export class KvTagLetter {
+	/** (required) Tag letter's color */
```

Comment:
make it explicitly required by adding a '!' after the variable name, i.e: `@Prop({ reflect: true }) color!: string;`

Reply:
same for any other required properties